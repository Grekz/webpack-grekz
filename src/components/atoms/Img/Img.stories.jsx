import React from 'react'

import { storiesOf } from '@storybook/react'
// import { action } from '@storybook/addon-actions'
// import { linkTo } from '@storybook/addon-links'

import Img from '.'

storiesOf('Img', module).add('empty', () => <Img alt="test" />)
