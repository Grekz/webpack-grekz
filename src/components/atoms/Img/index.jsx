import Img from './img.component'
export default Img
