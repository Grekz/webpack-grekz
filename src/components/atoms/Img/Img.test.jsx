import React from 'react'
import Img from './Img'
import { configure, shallow, mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'

configure({ adapter: new Adapter() })

describe('<Img /> rendering', () => {
  it('should render Img', () => {
    const wrapper = shallow(<Img />)
    expect(wrapper).toHaveLength(1)
  })
  it('should contain 1 img element', () => {
    const wrapper = shallow(<Img />)
    expect(wrapper.find('img')).toHaveLength(1)
  })
  it('mathces the snapshot', () => {
    const tree = shallow(<Img />)
    expect(toJson(tree)).toMatchSnapshot()
  })
})
