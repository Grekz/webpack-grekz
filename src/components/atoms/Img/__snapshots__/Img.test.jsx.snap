// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Img /> rendering mathces the snapshot 1`] = `<img />`;
